node/network/statement-distribution/src/lib.rs

```diff
@@ -874,14 +890,19 @@ async fn run(
 			}
 			FromOverseer::Signal(OverseerSignal::Conclude) => break,
 			FromOverseer::Communication { msg } => match msg {
-				StatementDistributionMessage::Share(relay_parent, statement) =>
+				StatementDistributionMessage::Share(relay_parent, statement) => {
+					inform_statement_listeners(
+						statement.clone(),
```

Comment:
If it's cloned inside the function should we pass `statement` there as a reference to save us one `clone`?